lib/api/manipulation.js

```diff
@@ -593,6 +602,9 @@ exports.insertAfter = function (target) {
  *   //      <li class="pear">Pear</li>
  *   //    </ul>
  *
+ * @param {string | Element | Element[] | Cheerio | Function} content - HTML
```

Comment:
Not sure how to do this properly, but we should use a [repeatable param](https://jsdoc.app/tags-param.html#multiple-types-and-repeatable-parameters) here.

```suggestion
 * @param {...(string | Element | Element[] | Cheerio | Function)} content - HTML
```